src/GeneratorState.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef GENSET_WHISPERPOWER_DDC_GENERATOR_STATE_HPP
+#define GENSET_WHISPERPOWER_DDC_GENERATOR_STATE_HPP
+
+#include <base/Time.hpp>
+#include <genset_whisperpower_ddc/GeneratorStatus.hpp>
+
+namespace genset_whisperpower_ddc {
+    struct GeneratorState {
+        base::Time time;
+
+        int rpm;
+        int udc_start_battery;
```

Comment:
What's "udc" here ?

Reply:
I think the "U" is for voltage (common symbol for voltage). Rename to "start_battery_voltage"